scripts/launch-s0-instance.sh

```diff
@@ -223,6 +224,8 @@ usage()
     echo "                                This shard is intended for use with backups of file systems, streams, and any other data that isn't organized"
     echo "                                into fixed size blocks."
     echo
+    echo "  -o | --on-demand          : Optional. Create an on-demand instance instead of the default spot instance."
```

Comment:
- This option costs Elastio money.  We shouldn't make it easy to type.
- Short option is easy to overlook.
- It breaks the [principle of least surprise][POLA]: `-o` is often used for `--output`.

```suggestion
    echo "       --on-demand          : Optional. Create an on-demand instance instead of the default spot instance."
```

[POLA]: https://en.wikipedia.org/wiki/Principle_of_least_astonishment

Reply:
I would go further and say it should be `--burn-money`

Reply:
)))